mappings/net/minecraft/inventory/SidedInventory.mapping

```diff
@@ -1,11 +1,15 @@
 CLASS net/minecraft/class_1278 net/minecraft/inventory/SidedInventory
-	METHOD method_5492 canInsertInvStack (ILnet/minecraft/class_1799;Lnet/minecraft/class_2350;)Z
+	COMMENT A special inventory interface for inventories that expose their contents to access via automation. i.e. The Hoopers.
```

Comment:
"Hoppers"

Reply:
This comment could do with some work, doesnt really say what it does tbh.

Reply:
```suggestion
	COMMENT A special inventory interface for inventories that expose different slots for different sides, such as Hoppers.
```